osbrain/tests/common.py

```diff
@@ -6,6 +6,10 @@
 from osbrain.helper import sync_agent_logger
 
 
+def receive(agent, message, topic=None):
+    agent.received.append(message)
```

Comment:
I like the idea of moving this function to `common`. We could, however, use a more explicit name? In example: `set_received`, `append_received` or something like that. `receive` seems a bit vague.

Maybe even `set_received_handler`, `append_received_handler`...

What you think?